.github/workflows/release.yaml

```diff
@@ -17,20 +14,13 @@ jobs:
     steps:
       - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v2.5.0
 
-      - name: Check if tag already exists
-        # note: this will fail if the tag already exists
-        run: |
-          [[ "${{ github.event.inputs.version }}" == v* ]] || (echo "version '${{ github.event.inputs.version }}' does not have a 'v' prefix" && exit 1)
-          git tag ${{ github.event.inputs.version }}
-
       - name: Check static analysis results
         uses: fountainhead/action-wait-for-check@297be350cf8393728ea4d4b39435c7d7ae167c93 # v1.1.0
         id: static-analysis
         with:
           token: ${{ secrets.XEOL_GITHUB_TOKEN }}
           # This check name is defined as the github action job name (in .github/workflows/testing.yaml)
           checkName: "Static analysis"
-          ref: ${{ github.event.pull_request.head.sha || github.sha }}
```

Comment:
any reason to remove these?

Reply:
The workflow is triggered by push event, so `github.event.pull_request.head.sha` is unnecessary.
And the default value is `github.sha`.

Reply:
ah great, I didn't know this was the default, thank you.